.github/workflows/publish.yaml

```diff
@@ -12,6 +12,7 @@ jobs:
         go-version: [1.15.x]
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
+
```

Comment:
This cannot be merged until we add the `permissions` section, please see: https://github.com/openfaas/ingress-operator/blob/master/.github/workflows/publish.yaml#L11

Reply:
Only change the publish step though, as the build should not do any pushes of images, so doesn't need write access to packages.

Reply:
nice, this is a new thing, i was not aware 😄 

done, PTAL